Makefile

```diff
@@ -11,7 +11,7 @@ TESTTAGS ?= ""
 test:
 	echo "mode: count" > coverage.out
 	for d in $(TESTFOLDER); do \
-		$(GO) test -tags $(TESTTAGS) -v -covermode=count -coverprofile=profile.out $$d > tmp.out; \
+		$(GO) test $(TESTTAGS) -v -covermode=count -coverprofile=profile.out $$d > tmp.out; \
```

Comment:
`$(GO) test -tags "$(TESTTAGS)" -v ...`
right?

Reply:
oh, I try to fix the error showed in the action summary page, and move the `-tags` string to the action settings, but it seems not work..
```
test-tags: ['', '-tags nomsgpack', '-tags "sonic avx"', '-tags go_json']
```
should this be moved back?